treeshr/treeshrp.h

```diff
@@ -370,21 +370,13 @@ typedef struct tag_info
 typedef struct tree_header
 {
   char version; /* Version of tree file format */
-#ifdef _AIX
```

Comment:
Can you call this out in the PR message?

Reply:
done